### DISCUSSION TITLE
Unable to Install 2.5.5 in Ubuntu 16.04

### DISCUSSION THREAD

**Discussion by lekemula** [rbenv/ruby-build#1841] [Build failures]
rbenv install 2.5.5 
Downloading ruby-2.5.5.tar.bz2...
-> https://cache.ruby-lang.org/pub/ruby/2.5/ruby-2.5.5.tar.bz2
Installing ruby-2.5.5...

BUILD FAILED (Ubuntu 16.04 using ruby-build 20190423-2-gd27e745)

Inspect or clean up the working tree at /tmp/ruby-build.20190515142530.25720
Results logged to /tmp/ruby-build.20190515142530.25720.log

Last 10 log lines:
random.o: In function `explicit_bzero':
/usr/include/x86_64-linux-gnu/bits/string_fortified.h:83: undefined reference to `__explicit_bzero_chk'
/usr/include/x86_64-linux-gnu/bits/string_fortified.h:83: undefined reference to `__explicit_bzero_chk'
/usr/include/x86_64-linux-gnu/bits/string_fortified.h:83: undefined reference to `__explicit_bzero_chk'
/usr/include/x86_64-linux-gnu/bits/string_fortified.h:83: undefined reference to `__explicit_bzero_chk'
/usr/include/x86_64-linux-gnu/bits/string_fortified.h:83: undefined reference to `__explicit_bzero_chk'
random.o:/usr/include/x86_64-linux-gnu/bits/string_fortified.h:83: more undefined references to `__explicit_bzero_chk' follow
collect2: error: ld returned 1 exit status
Makefile:231: recipe for target 'miniruby' failed
make: *** [miniruby] Error 1

**Comment by ultrasam** [comment]
Could you please share the log file?

**Comment by lekemula** [comment]
@ultrasam Sure, please find it in this [link](https://pastebin.com/AvjDjKmB)

**Comment by mislav** [comment]
@lekemula Thank you for reporting. So weird! I haven't seen such a build error before.
